kairon/api/app/routers/pos.py

```
@@ -105,16 +105,44 @@ def list_pos_orders(
 
 
 @router.post("/pos_order", response_model=Response)
-def create_order(req: POSOrderRequest, session_id: str = Query(...),
+async def create_order(req: POSOrderRequest, session_id: str = Query(...),
                  current_user: User = Security(Authentication.get_current_user_and_bot, scopes=ADMIN_ACCESS)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)